src/Modules/Settings/Dnn.PersonaBar.Prompt/Prompt.Web/src/components/Output/Command.jsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import Parser from "html-react-parser";
+import Localization from "localization";
+import {sort} from "../../helpers";
+
+const Command = ({commandList, IsPaging}) => {
```

Comment:
If you'd just rename this to plural which this component represents. `Commands` or `CommandList`?